plugins/explore/api-report.md

```diff
@@ -31,14 +30,17 @@ export const catalogEntityRouteRef: ExternalRouteRef<
   true
 >;
 
-// @public (undocumented)
-export const DomainCard: (props: { entity: DomainEntity }) => JSX.Element;
```

Comment:
What happened here? :grin: is the new card supposed to be exported or not? Who would be using this card already? This plugin has pretty wide usage, I think it likely makes sense to just deprecate it to begin with to avoid a breaking change?

Reply:
Yeah, we use this page directly in a custom `ExplorePage` in our `packages/app`. I think we could use the `EntityExplorePage kind=domain` here, but it would be a breaking change.

Reply:
I reinstated the DomainCard for backwards compatability